src/api/app/views/webui/attribute/_issue_fields.html.haml

```diff
@@ -0,0 +1,6 @@
+.nested-fields
+  .field
+    = f.select :issue_tracker_id, IssueTracker.all.collect { |tracker| [ tracker.name, tracker.id ] }.sort, include_blank: false
```

Comment:
Yes. My point was only converting it automatically... not refactoring it. 
So, I will not move code.